src/main/java/Event.java

```diff
@@ -0,0 +1,19 @@
+public class Event extends Task {
+    protected final String from;
+    protected final String to;
+
+    public Event(String input) {
+        super(input.split("/", 3)[0].trim());
+
+        String[] parts = input.split("/", 3);
+
+        this.from = parts[1].trim().substring(5); // ignore "from "
```

Comment:
May I ask why you chose to use inline comments instead of block comments? Nontheless, I like that your comments are short :)